example/properties_directory/non-scalar/hubbard_parameters.json

```diff
@@ -0,0 +1,37 @@
+{
+    "name": "hubbard_parameters",
+    "units": "eV",
+    "values": [
+        {
+            "category": "hubbard_u",
+            "headers": ["Site no.", "Atomic species", "Orbital/manifold", "U (eV)"],
```

Comment:
These headers already assume extraction from a QE calculation - we should not make a connection here yet. This is a data structure for the property only. I'd remove headers